src/main/resources/templates/chapters.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.thymeleaf.org">
+<head>
+    <meta charset="UTF-8">
+    <title>Chapters</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1" >
+    <link rel="stylesheet" href="static/common.css" th:href="@{static/common.css}">
+    <link rel="stylesheet" href="static/chapters.css" th:href="@{static/chapters.css}">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+</head>
+<body>
+
+<div th:insert="fragments/header :: header"></div>
+
+<div th:insert="fragments/navbar :: navbar('TODO', 'TODO', 'TODO', 'TODO')"></div>
+
+<div th:insert="fragments/hero :: hero(#{page_title}, #{page_info})"></div>
+
+<div th:insert="fragments/searchbar :: searchbar"></div>
+
+<div class="chapter-download-card">
+    <h2 class="chapter-download-card__book-name"
+    th:text="${'replace with book name from book object'}">{Book Name}</h2>
+    <a class="chapter-download-card__button"
+        th:href="${'/'+ 'replace this with url from book object'}">
+        <em class="material-icons">get_app</em>
+        <p >Download Book</p>
+    </a>
+</div>
+
+<div class="l-list-container" >
+    <div th:each="language: ${languageList}">
+        <div th:insert="fragments/chaptercard :: chaptercard(${language.chapterNumber}, ${language.localizedName}, ${language.url}, '#')"></div>
```

Comment:
Did you forget to update the variable?

Reply:
oh yes, my fault. i was using that to test the template.